README.rst

```diff
@@ -21,19 +21,10 @@ Virtual environment
 
 To create a working virtual environment::
 
-    virtualenv -p 3.8 venv  # Last Python version supported by serverless-azure-functions
+    virtualenv -p 3.11 venv
```

Comment:
why not got with 3.12 ? Just asking

Reply:
3.11 is the latest supported for now: https://learn.microsoft.com/en-us/azure/azure-functions/supported-languages?tabs=isolated-process%2Cv4&pivots=programming-language-python#languages-by-runtime-version